www/docs/src/content/docs/customizing/studiocms-ui/components/select-searchable.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Select (Searchable)
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import PreviewCard from '~/components/PreviewCard.astro';
+import { SearchSelect } from '@studiocms/ui/components';
+
+A variation of the `<Select />` element that supports arrow keys and filtering.
+
+**Can be used as a drop-in replacement.**
+
+## Usage
+
+<Tabs>
+  <TabItem label="Preview">
+    <PreviewCard>
+      <SearchSelect
+        label='Select Element'
+        options={[
+          { label: "Option 1", value: "opt-1-1" },
+          { label: "Option 2", value: "opt-2-1" },
+          { label: "Option 3", value: "opt-3-1" },
+        ]}
+      />
+    </PreviewCard>
+  </TabItem>
+  <TabItem label="Code">
+    ```astro
+    ---
+    import { SearchSelect } from '@studiocms/ui/components';
+    ---
+
+    <SearchSelect
+      label='Searchable Select Element'
+      options={[
+        { label: "Option 1", value: "opt-1" },
+        { label: "Option 2", value: "opt-2" },
+        { label: "Option 3", value: "opt-3" },
+      ]}
+    />
+    ```
+  </TabItem>
+</Tabs>
+
+### Default Value
+
+You can set a default value using the `defaultValue` prop:
+
+<Tabs>
+  <TabItem label="Preview">
+    <PreviewCard>
+      <SearchSelect
+        label='Searchable Select Element'
+        defaultValue='opt-1-1'
+        options={[
+          { label: "Option 1", value: "opt-1-1" },
+          { label: "Option 2", value: "opt-2-1" },
+          { label: "Option 3", value: "opt-3-1" },
+        ]}
+      />
+    </PreviewCard>
+  </TabItem>
+  <TabItem label="Code">
+    ```astro
+    ---
+    import { SearchSelect } from '@studiocms/ui/components';
+    ---
+
+    <SearchSelect
+      label='Searchable Select Element'
+      defaultValue='opt-1'
+      options={[
+        { label: "Option 1", value: "opt-1" },
+        { label: "Option 2", value: "opt-2" },
+        { label: "Option 3", value: "opt-3" },
+      ]}
+    />
+    ```
+  </TabItem>
+</Tabs>
+
+### Form Support
+
+You can use the `isRequired` prop to mark the select input as required. This will also add a red `*` to
+the label.
+
+If you need to access the element's value later on, you can pass the `name` prop. It will then appear in any FormData
+you construct.
```

Comment:
```suggestion
If you need to access the element's value later on, you can pass the `name` prop. It will then appear in any FormData
you construct. If no name is set, a random one is generated for the component.
```